src/chromium/interfaces/listeners.js

```diff
@@ -19,7 +41,27 @@ export function initPlatformListeners() {
   });
 
   browser.runtime.onInstalled.addListener(async () => {
+    registerEventRules();
     await getIsFirefoxInstalled();
     browser.storage.sync.set({ currentExternalBrowser: "Firefox" });
   });
+
+  // From: https://github.com/Rob--W/crxviewer/blob/942b02fa871ff4ef6a29cf18b266711725314e86/src/background.js#L325-L341
+
+  //// Work-around for crbug.com/388231: onInstalled is not fired when the
```

Comment:
This bug cannot be reproduced any more according to the comments in the bug, so the work-around can probably be removed. I would mind keeping it here though, because the onStartup event is so infrequent.